src/NHibernate/Cache/IBatchableCacheConcurrencyStrategy.cs

```diff
@@ -19,29 +19,34 @@ namespace NHibernate.Cache
 	/// for a collection.
 	/// </para>
 	/// </remarks>
+	// 6.0 TODO: merge into ICacheConcurrencyStrategy
 	public partial interface IBatchableCacheConcurrencyStrategy : ICacheConcurrencyStrategy
 	{
 		/// <summary>
-		/// Attempt to retrieve multiple objects from the Cache
+		/// Attempt to retrieve multiple items from the cache.
 		/// </summary>
-		/// <param name="keys">The keys (id) of the objects to get out of the Cache.</param>
-		/// <param name="timestamp">A timestamp prior to the transaction start time</param>
-		/// <returns>An array of cached objects or <see langword="null" /></returns>
+		/// <param name="keys">The keys of the items.</param>
+		/// <param name="timestamp">A timestamp prior to the transaction start time.</param>
+		/// <returns>The cached items, matching each key of <paramref name="keys"/> respectively. For each missed key,
+		/// it will contain a <see langword="null" />.</returns>
 		/// <exception cref="CacheException"></exception>
 		object[] GetMany(CacheKey[] keys, long timestamp);
 
 		/// <summary>
-		/// Attempt to cache objects, after loading them from the database.
+		/// Attempt to cache items, after loading them from the database.
 		/// </summary>
-		/// <param name="keys">The keys (id) of the objects to put in the Cache.</param>
-		/// <param name="values">The objects to put in the cache.</param>
+		/// <param name="keys">The keys of the items.</param>
+		/// <param name="values">The items.</param>
 		/// <param name="timestamp">A timestamp prior to the transaction start time.</param>
-		/// <param name="versions">The version numbers of the objects we are putting.</param>
-		/// <param name="versionComparers">The comparers to be used to compare version numbers</param>
+		/// <param name="versions">The version numbers of the items.</param>
+		/// <param name="versionComparers">The comparers to be used to compare version numbers.</param>
 		/// <param name="minimalPuts">Indicates that the cache should avoid a put if the item is already cached.</param>
-		/// <returns><see langword="true" /> if the objects were successfully cached.</returns>
+		/// <returns>An array of boolean indicating if each item was successfully cached.</returns>
 		/// <exception cref="CacheException"></exception>
 		bool[] PutMany(CacheKey[] keys, object[] values, long timestamp, object[] versions, IComparer[] versionComparers,
 								  bool[] minimalPuts);
+
+		// 6.0 TODO: remove for using ICacheConcurrencyStrategy.Cache re-typed CacheBase instead
+		CacheBase CacheBase { get; }
```

Comment:
We can change this interface as it is not yet released.

This property is somewhat obsolete as soon as added. But it should not bother many implementors, since mapping does not allow to specify custom cache concurrency implementations.